### DISCUSSION TITLE
微軟與Nvidia發表全球最大含5,300億個參數的AI語言模型

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#128] [General]
自然語言處理可以應用的範圍廣泛，像是語音助理、搜尋建議更正......，微軟與Nvidia這次的合作克服了大型語言模型會遇到的障礙——放不下大量參數的GPU記憶體。他們結合Megatron-LM和PyTorch深度學習最佳化函式庫DeepSpeed，創建了高效且可擴展的3D平行系統，將資料、工作管線和基於張量切片的平行性結合在一起，來克服訓練大型語言模型所遭遇的困難。

順便附上一張電腦學習自然語言的進程

<img width="1003" alt="截圖 2021-10-24 下午9 03 14" src="https://user-images.githubusercontent.com/90544548/138595441-a0bec301-53f1-4bb1-8102-184cd6a63319.png">

[微軟與Nvidia發表全球最大含5,300億個參數的AI語言模型](https://ithome.com.tw/news/147225)

**Comment by Jie2222** [comment]
這樣的AI 可以同時做到回答問題、寫文章、語言翻譯等任務，但也被擔憂會用來製造假消息
